packages/cli/src/index.ts

```diff
@@ -446,6 +566,11 @@ yargs
           description: 'Project directory.',
           default: '.',
         },
+        config: {
```

Comment:
Does the `preview` command needs the `config` property? Should we add it in this MR?

Reply:
I think this was added during conflict resolution, `PreviewProjectOptions` contain the `config` option, so it should be there.

Reply:
Removed